backend/tests_configs.ini

```diff
@@ -98,6 +98,7 @@ preview.jpg.restricted_dims = True
 email.notification.activated = false
 website.base_url = http://localhost:6543
 color.config_file_path = %(here)s/color-test.json
+user.reset_password.validity = 5
```

Comment:
What is validity ? If a duration, then the name should explain it

Reply:
it's a duration, name should probably be renamed but clearly not in this PR.

Reply:
https://github.com/tracim/tracim/issues/970

Reply:
hm... this is a line you add in the current MR, why is it not the "place to rename this"?

Reply:
user.reset_password.validity is not a new param. it already exist in tracim config . I prefer to do it separately because that not really associated with neww response mecansim.